bakefiles/extra.docker-bake.json

```diff
@@ -1,6 +1,14 @@
 {
   "group": [
     {
+      "default": [
+        {
+          "targets": [
+            "geospatial-ubuntugis",
+            "geospatial-dev-osgeo"
+          ]
```

Comment:
This was done as a workaround because the `bake --print` command has a bug when using following more concise json. (docker/buildx#1058)

```json
"targets": [
  "ubuntugis",
  "osgeo"
]
```